src/Microsoft.Identity.Client/UIBehavior.cs

```diff
@@ -60,11 +60,29 @@ public partial struct UIBehavior
         public static readonly UIBehavior Never = new UIBehavior("attempt_none");
 #endif
 
-        internal string PromptValue { get; set; }
+        internal string PromptValue { get; }
 
         internal UIBehavior(string promptValue)
```

Comment:
Not changed in this PR, but does this constructor need to be _internal_? Looks like it could be _private_.